RoboJackets/robocup-software
# Launch file enhancements

## Description
basically just adds a bunch of different launch file configs. Long term we can build all and dynamically choose which ones to run using ifconditions, but for now this is better than commenting out nodes. 

To see how to use ifconditions, check out the use_grsim flag in sim.launch.py

Also sim2play will not use grSim anymore.

## Steps to test

Expected result: Can switch between different setups using the new make flags added in the makefile.
